lib/rubocop/cop/betterment/authorization_in_controller.rb

```diff
@@ -37,17 +37,17 @@ def initialize(config = nil, options = nil)
           super
           @unsafe_parameters = cop_config.fetch("unsafe_parameters").map(&:to_sym)
           @unsafe_regex = Regexp.new cop_config.fetch("unsafe_regex")
-          @param_wrappers = []
+        end
+
+        def on_new_investigation
+          super
+          @class_methods = {}.freeze
+          @param_wrappers = [].freeze
```

Comment:
__comment:__ I changed this to freeze since it actually exposed a few bugs in other code that was silently mutating it's arguments. I left it in to prevent future similar regressions.